src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -0,0 +1,140 @@
+package com.googleinterns.zoomtube.servlets;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.googleinterns.zoomtube.data.AuthenticationStatus;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+
+public class AuthenticationServletTest {
+  private AuthenticationServlet servlet;
+  private MockHttpServletRequest request;
+  private MockHttpServletResponse response;
+
+  @Before
+  public void setUp() throws ServletException {
+    servlet = new AuthenticationServlet();
+    servlet.init();
+    request = new MockHttpServletRequest();
+    response = new MockHttpServletResponse();
+  }
+
+  @Test
+  public void doGet_loggedIn_true() throws ServletException, IOException {
```

Comment:
Can you add comments to the tests?

Reply:
You do not typically need comments unless something is not obvious from reading the code. Was there something you thought needed a comment?